pkg/grpc/server.go

```diff
@@ -69,11 +71,18 @@ func NewServersFromConfigurationAndServe(configurations []*configuration.ServerC
 
 		// Enable TLS transport credentials if provided.
 		hasCredsOption := false
-		if tlsConfig, err := util.NewTLSConfigFromServerConfiguration(configuration.Tls, requestTLSClientCertificate); err != nil {
-			return err
-		} else if tlsConfig != nil {
+		switch transportSecurity := configuration.GetTransportSecurity().(type) {
```

Comment:
Please use `.TransportSecurity` instead of `.GetTransportSecurity()`. In the Buildbarn code base we only use `.Get*()` when the parent may potentially be nil. In this case it's guaranteed to be non-nil.